jerry-core/lit/lit-char-helpers.c

```diff
@@ -289,32 +289,32 @@ lit_char_hex_to_int (ecma_char_t c) /**< code unit, corresponding to
  * @return true if decoding was successful, false otherwise
  */
 bool
-lit_read_code_point_from_hex (lit_utf8_byte_t *buf_p, /**< buffer with characters */
-                              lit_utf8_size_t number_of_characters, /**< number of characters to be read */
-                              lit_code_point_t *out_code_point_p) /**< [out] decoded result */
+lit_read_code_unit_from_hex (lit_utf8_byte_t *buf_p, /**< buffer with characters */
+                             lit_utf8_size_t number_of_characters, /**< number of characters to be read */
+                             ecma_char_ptr_t out_unit_point_p) /**< [out] decoded result */
 {
-  lit_code_point_t code_point = 0;
+  ecma_char_t code_unit = LIT_BYTE_NULL;
```

Comment:
Sorry, for keeping on amending my own reviews, but I just had a closer look at lit-char-helpers.h and found `LIT_CHAR_NULL` therein. Seems to fit here perfectly.